### PR TITLE
Mag 42 - Double check DHL Enabled at the Default and Store levels

### PR DESCRIPTION
**Pull request by BronzePointKrel**
Previously MAG-34

### CODE REVIEW

File: view/frontend/web/template/shipping.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
 
                 <render args="shippingMethodListTemplate"/>
                 
-                <div class="reach-tax-duty" data-bind="fadeVisible: dutyCharges() > 0 && getReachEnabled() != 0">
+                <div class="reach-tax-duty" data-bind="fadeVisible: dutyCharges() > 0 && getReachEnabled() != 0 && getDHLEnabled() != 0">
```
Comment (lizzyWithReach) [inline view/frontend/web/template/shipping.html]: the only issue here is that the duties and taxes checkbox should show up if DHL is enabled, but it doesn't matter if Reach is enabled. They should work independently.

Comment (lizzyWithReach) [reply view/frontend/web/template/shipping.html]: (by "they" I mean the Reach vs DHL functionality)

**Review by lizzyWithReach — CHANGES_REQUESTED** [review]

File: view/frontend/web/template/shipping.html (hunk 1 above)
Comment (suraiyavic) [reply view/frontend/web/template/shipping.html]: I do not know enough about the business rules to check/comment about the correctness here. So I would go with what Lizzy said.

I have to read/revisit the older meeting notes available in our system. But would appreciate if someone points me to any better source of information that they can think of/know of. Thanks.

**Review by suraiyavic — COMMENTED** [review]
Actually as I think about it Lizzy's comment about one independent (duty label can be enabled even if reach is not) of the other makes sense.

File: view/frontend/web/template/shipping.html (hunk 1 above)
Comment (BronzePointKrel) [reply view/frontend/web/template/shipping.html]: Updated code to reflect requested change.

**Comment by BronzePointKrel** [comment]
I believe all issues are now resolved.

**Review by lizzyWithReach — APPROVED** [review]

**Pull request merged**